README.md

```diff
@@ -50,31 +112,6 @@ seattle_cdp_parsed_events = seattle_scraper.get_events(
 )
 ```
 
-### Scrapers
-
-In-progress or completed scrapers.
-
--   [cdp_scrapers.instances.seattle.get_events](https://councildataproject.org/cdp-scrapers/cdp_scrapers.instances.html#module-cdp_scrapers.instances.seattle)
-
-If you would like to deploy a CDP instance or would like to use this library as
-a method for retrieving formatted legislative data, please feel free to contribute
-a new custom municipality scraper!
-
-#### Custom Scrapers
-
-If it isn't possible to use our generalized Legistar tooling to write your scraper,
-we welcome the addition of custom scrapers, however please see our documentation
-on the
-[minimum data required for CDP event ingestion](https://councildataproject.org/cdp-backend/ingestion_models.html).
-
-From there, begin with our
-[empty custom scraper function template](https://councildataproject.org/cdp-scrapers/_modules/cdp_scrapers/instances/empty.html#get_events) and fill in your scraper.
-
-## Installation
-
-**Stable Release:** `pip install cdp-scrapers`<br>
-**Development Head:** `pip install git+https://github.com/CouncilDataProject/cdp-scrapers.git`
-
```

Comment:
Keep this Installation section please. And maybe even move it up next to the quickstart.

Reply:
Oops yeah agree and that may have been my finger drifting off by accident.